twilight-model/src/gateway/event/dispatch.rs

```diff
@@ -15,9 +15,9 @@ use serde::{
 #[serde(untagged)]
 pub enum DispatchEvent {
     AutoModerationActionExecution(AutoModerationActionExecution),
-    AutoModerationRuleCreate(Box<AutoModerationRuleCreate>),
-    AutoModerationRuleDelete(Box<AutoModerationRuleDelete>),
-    AutoModerationRuleUpdate(Box<AutoModerationRuleUpdate>),
+    AutoModerationRuleCreate(AutoModerationRuleCreate),
+    AutoModerationRuleDelete(AutoModerationRuleDelete),
+    AutoModerationRuleUpdate(AutoModerationRuleUpdate),
```

Comment:
Is there a reason as to why this was included in this pr?

Reply:
Since the threshold for the dispatch payload was increased these boxed types now fit under the threshold so the boxing was removed.